File: src/org/jetbrains/agda/core/CTypeSignature.java
package org.jetbrains.agda.core;

/**
 * @author Evgeny.Kurbatsky
 */
public class CTypeSignature extends CDeclaration {
    private String myName;

    public CTypeSignature(String name, CExpression type) {
        this.myName = name;
        this.myType = type;
    }



    @Override
    public String toString() {
        return myName + " : " + myType;
    }
}

File: src/org/jetbrains/agda/core/CTypeSignature.kt
package org.jetbrains.agda.core

/**
 * @author Evgeny.Kurbatsky
 */
public open class CTypeSignature(name : String, aType : CExpression) : CDeclaration(aType) {
    private var myName : String? = null
    public open fun getName() : String? {
        return myName
    }
    public fun toString() : String? {
        return myName + " : " + aType
    }
    {
        this.myName = name
    }

}
